feeds/pypi/pypi.go

```diff
@@ -69,11 +68,19 @@ func fetchPackages() ([]*Package, error) {
 	return rssResponse.Packages, nil
 }
 
-type Feed struct{}
+type Feed struct {
```

Comment:
I think the addition of the `Feed` struct to abstract the url is worthy of being a separate commit, but that's only a preference

Reply:
Agreed :+1: